lib/endpoints/class-wp-json-posts-controller.php

```diff
@@ -9,6 +9,76 @@ public function __construct( $post_type ) {
 	}
 
 	/**
+	 * Register the routes for the objects of the controller
+	 * 
+	 */
+	public function register_routes() {
+
+		parent::register_routes();
```

Comment:
The idea was more to potentially have some DRY code in the base for things like schema endpoints, but after thinking on that I don't think we can do it automajically very well anyway